src/purchase-selection/utils.ts

```diff
@@ -88,22 +88,16 @@ export const getDefaultUserProfiles = (
   input: PurchaseSelectionBuilderInput,
   product: PreassignedFareProduct,
 ): UserProfileWithCount[] => {
-  const selectableProfiles = getSelectableUserProfiles(
-    input.userProfiles,
-    product,
-  );
-  const profile = selectableProfiles.reduce((selected, current) =>
-    current.userTypeString === input.defaultUserTypeString ? current : selected,
-  );
+  const profile = input.userProfiles
```

Comment:
Not sure how to refactor this one to work with the new hook, any suggestions @gorandalum ?

Reply:
If we didn't want to touch this code we could let `getSelectableUserProfiles` still exist, and make `useSelectableUserProfiles` use `getSelectableUserProfiles`. But no biggie, this looks ok!